src/client/components/MessageForm/MessageForm.jsx

```diff
@@ -0,0 +1,31 @@
+import { useForm } from 'react-hook-form';
+import styles from './message-form.module.css';
+
+export const MessageForm = ({ send }) => {
+  const { handleSubmit, register, reset } = useForm();
+
+  const onSubmit = (data) => {
+    send(data);
+    reset();
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className={styles['message-form']}>
+      <div className={styles['message-form__field']}>
+        <label htmlFor="message" className="visually-hidden">
+          Message
+        </label>
```

Comment:
The label for the message input is visually hidden, which is good for accessibility. Ensure that the `visually-hidden` class is correctly defined in your CSS to maintain accessibility standards.